CMakeLists.txt

```diff
@@ -145,6 +145,7 @@ if (ENABLE_VECTOR_PREDICATE_SHORT_CIRCUIT)
   )
 endif()
 
+option(ENABLE_SINGLE_NODE_SERVER "Enables server mode in single node setting" ON)
```

Comment:
Yes, for now `Quickstep` does not declare the `grpc` dependency. So, by default, we don't build any binaries that link to `grpc`.